src/components/AddItemForm.tsx

```diff
@@ -1,22 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+interface AddItemFormsProps {
+  addTask: (task: string, newKey: string) => void;
+}
+function AddItemForm({ addTask }: AddItemFormsProps) {
+  const [inputValue, setInputValue] = useState<string>('');
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (inputValue) {
```

Comment:
nao ta errado, mas o jeito mais explicito de fazer esse check seria
```suggestion
    if (inputValue.length > 0) {
```